sources/sql_database/schema_types.py

```diff
@@ -134,6 +178,12 @@ def row_tuples_to_arrow(
 
     arrow_schema = columns_to_arrow(columns, tz=tz)
 
+    columns_list = list(columns.values())
+    included_columns_idx = [
```

Comment:
are you removing columns with not recognized data types? why? we should keep them. `dlt` will later try to map missing columns into dlt schema (in extract). so it should work